spec/models/intake_spec.rb

```diff
@@ -1049,38 +1050,71 @@
       end
     end
   end
+  describe "#update_or_create_13614c_document" do
+    before do
+      example_pdf = Tempfile.new("example.pdf")
+      example_pdf.write("example pdf contents")
+      allow(intake).to receive(:pdf).and_return(example_pdf)
+    end
 
-  describe "#update_or_create_14446_document" do
+    let(:intake) { create(:intake) }
+
+    context "when there is not an existing 13614-C document" do
```

Comment:
This test is not strictly doc-type-specific, so the context could be renamed `"when there is no doc of the given type for the client"`. Not a blocker.